src/pages/components/CertificateForm.js

```diff
@@ -130,14 +132,35 @@ export default function CertificateForm(props) {
   return (
     <>
       <TimelineWrapper area="timeline">
-        <Timeline {...props}>
-          {props.items.map(({ message, ...rest }) => (
-            <Timeline.Item key={rest.title} {...props} {...rest}>
-              {message && <p>{message}</p>}
-            </Timeline.Item>
-          ))}
+        <Timeline
+          name={dataChain?.id && persona.company}
+          disclaimer={disclaimer}
+          variant={'hyproof'}
+        >
+          <Timeline.Item
+            variant="hyproof"
+            title={'Initiation'}
+            checked={dataFinal?.id === 'initiated'}
+          >
+            {dataFinal?.state === 'initiated' ? dataFinal.created_at : null}
+          </Timeline.Item>
+          <Timeline.Item
+            variant="hyproof"
+            title={'Carbon Embodiment'}
+            checked={dataChain?.embodied_co2}
+          >
+            {persona.embodied_co2 && formatTimelineDate(dataFinal.updated_at)}
```

Comment:
This looks amazing .. i can see the timeline ( changing starting from the upper part and going down ) .. 

Just a small thing .. where are you saving _embodied_co2_ ? is this part of the `Context.js` or part of `App.js` `personas` ?

Also is this meant to work in the second step of the flow ? I am getting that `ErrorFoundCertHasWrongHash` error .. maybe i should show it to you ..

Reply:
Component already has it from a context